Maps/ArchipelagoCampaign/HotS/ap_phantoms_of_the_void.SC2Map/Objects

```diff
@@ -4882,7 +4882,7 @@
         <GroupObject Id="1306"/>
         <GroupObject Id="1322"/>
     </Group>
-    <Group Type="ObjectUnit" Name="Brutal Removal" Id="4">
+    <Group Type="ObjectUnit" Name="Starting Zerg " Id="4">

```

Comment:
Why the rename?

Reply:
Wasn't supposed to be a rename, not sure why this happened. Or why there's a trailing space. Fixing